src/benchmarks/micro/corefx/System.Globalization/Perf.StringCultureSpecific.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+
+namespace System.Globalization.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.CoreCLR)]
+    public class Perf_StringCultureSpecific
+    {
+        private string _value, _same, _diffAtFirstChar, _diffAtLastChar, _firstHalf, _secondHalf;
+
+        public static IEnumerable<(CultureInfo CultureInfo, CompareOptions CompareOptions)> GetCultureOptions()
+        {
+            // Ordinal and OrdinalIgnoreCase use single execution path for all cultures, so we test it only for "en-US"
+            yield return (new CultureInfo("en-US"), CompareOptions.Ordinal);
+            yield return (new CultureInfo("en-US"), CompareOptions.OrdinalIgnoreCase);
+
+            yield return (new CultureInfo("en-US"), CompareOptions.None);
+            yield return (new CultureInfo("en-US"), CompareOptions.IgnoreCase);
```

Comment:
>CultureInfo("en-US"), CompareOptions.IgnoreCase); [](start = 30, length = 49)

could you please add a case for ignore non space option too?

Reply:
good point, done!